### ISSUE TITLE
Redundant check, might indicate bug

### ISSUE THREAD

**Issue by dominikh** [ttacon/libphonenumber#26]
The following piece of code checks for the same condition twice. The second condition is either redundant, or incorrect:

```
if len(nationalPrefix) == 0 || len(nationalPrefix) == 0 {
    // If the region doesn't have a national prefix at all,
    // we can safely return the national format without worrying
    // about a national prefix being added.
    formattedNumber = nationalFormat
    break
}
```

```
/src/github.com/ttacon/libphonenumber/phonenumberutil.go:1400:6: identical expressions on the left and right side of the '||' operator
```

**Comment by ttacon** [comment]
Nice catch, I'll correct this.

**Comment by ttacon** [comment]
Closed via #75
